yarn/src/main/scala/org/apache/spark/deploy/yarn/YarnAllocator.scala

```diff
@@ -435,19 +437,24 @@ private[yarn] class YarnAllocator(
         // there are some exit status' we shouldn't necessarily count against us, but for
         // now I think its ok as none of the containers are expected to exit
         if (completedContainer.getExitStatus == ContainerExitStatus.PREEMPTED) {
-          logInfo("Container preempted: " + containerId)
+          val msg = "Container preempted: " + containerId
```

Comment:
prefer string interpolation: `s"Container preempted: $containerId"`.  (I know it was that way before, perhaps from before scala 2.10 or it just slipped through, but in any case worth fixing now ...)